Sunrise/Cells/CartItemCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CartItemCell.swift
+//  Sunrise
+//
+//  Created by Nikola Mladenovic on 5/31/16.
+//  Copyright © 2016 Commercetools. All rights reserved.
+//
+
+import UIKit
+
+class CartItemCell: UITableViewCell {
```

Comment:
@lauraluiz yes, we can rename it to `CartLineItemCell`. This class contains properties which represent the UI element (in this case labels and an image view).